eskip/parser.y

```diff
@@ -137,28 +137,28 @@ route:
 			lbAlgorithm: $5.lbAlgorithm,
 			lbEndpoints: $5.lbEndpoints,
 		}
-		$1.matchers = nil
+		$1.predicates = nil
 		$3.filters = nil
 		$5.lbEndpoints = nil
 	}
 
-frontend:
-	matcher {
-		$$.matchers = []*matcher{$1.matcher}
+predicates:
```

Comment:
This renames rule and opens a way to parse predicates and filters separately, see draft branch https://github.com/zalando/skipper/tree/eskip/separate-parse-predicates-filters based on this